grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -567,19 +580,29 @@ class QueryService implements Status {
         issue
     }
 
+    Boolean areLinksApproved(String projectKey, String consentKey) {
+        List<ConsentCollectionLink> links = ConsentCollectionLink.findAll("from ConsentCollectionLink as ccl where ccl.projectKey= :projectKey and ccl.consentKey = :consentKey",
```

Comment:
I don't recommend GORM style queries. This can easily be a `getSqlConnection()` call that just queries for the status.